### ISSUE TITLE
SAP UI5 SDK ist not accesible

### ISSUE THREAD

**Issue by ChristopherMoltisanti1993** [SAP/ui5-language-assistant#662]
Hi, 

I'm trying to use this extension, but unfortunately I am receiving the following error:
![image](https://github.com/SAP/ui5-language-assistant/assets/71293531/f3cb5d0c-760a-4446-8eb6-3f58553b5244)
In manifest.json **minUIVersion** ist set to:
![image](https://github.com/SAP/ui5-language-assistant/assets/71293531/214dddfe-d7dd-4eab-a308-5010427a3b47)

I am using corporate proxy and found out that there was an issue with it, but according to this it should be fixed:
https://github.com/SAP/ui5-language-assistant/issues/285

Am I missing something:)? Need some additional steps need to be proceed to make this working?

**Comment by mr25mr** [comment]
@ChristopherMoltisanti1993 
can you please verify that
* `no_proxy` param is NOT set in `env`
*  `http_proxy` or `https_proxy` param is set in `env` as per your need (http vs https)
For details see this comment: https://github.com/SAP/ui5-language-assistant/issues/285#issuecomment-738118315

If above points does not solves your problem, will **Offline work** mood be an option for you? For this see instructions. [2. Offline work](https://blogs.sap.com/2023/07/04/sapui5-language-assistant-boosts-developer-productivity-with-building-blocks-offline-work-and-more/)

**Comment by ChristopherMoltisanti1993** [comment]
@mr25mr 
Sorry for my ignorance, but could you please tell me where I am supposed to look for ENV. Also where should **https_proxy** param be set?

**Comment by ChristopherMoltisanti1993** [comment]
Hi @mr25mr,

Isn't it risky to manipulate those Node settings? I am working on computer wtih no admin rights, so when I brake NodeJS it will be hard to restore it without IT help :) 

But I've done something else which is very strange. I've set up environment for Offline Work and all is working correctly with SAPUI5WebServer. I've set up 1.96.24 and 1.96.25 SAPUI5 version for Offline Work. 

And then i checked it one more time Online Mode and for this 2 versions it seems to work. Of course I've restored all the setting to work in Online Mode, so I've deleted localhost in extension settings:
![image](https://github.com/SAP/ui5-language-assistant/assets/71293531/83d6476d-db84-4653-89d2-b87e0b589e4e)
I've even reinstalled extension, and I've deleted all the downloaded SDKs, just to make sure. Isn't it strange? :)
